format/index.md

```diff
@@ -233,31 +241,32 @@ Relationships may be to-one or to-many.
 The value of a relationship **MUST** be an object (a "relationship object"),
 which **MUST** contain at least one of the following:
 
-* `links`: a member that contains at least one of the following:
-  * `self`: a member, whose value is a URL for the relationship itself (a
-    "relationship URL"). This URL allows the client to directly manipulate the
+* `links`: a member that contains at least one of the following [links]:
+  * `self`: a member, whose value is a link for the relationship itself (a
+    "relationship link"). This link allows the client to directly manipulate the
     relationship. For example, it would allow a client to remove an `author`
     from an `article` without deleting the `people` resource itself.
-  * `related`: a member, whose value is a related resource URL, as defined below.
+  * `related`: a member, whose value is a "related resource link", as defined
+    below.
 * `data`: a member, whose value represents "resource linkage" (defined below).
 * `meta`: a member that contains non-standard meta-information about the
   relationship.
 
 A relationship object that represents a to-many relationship **MAY** also contain
-pagination links under the `links` member, as described below.
+[pagination] links under the `links` member, as described below.
 
```

Comment:
Do you think a "**Related Resource Links**" sub-heading here would be helpful for organization and cross-linking? Adding one might also mean adding a "**Resource Linkage**" sub-heading near line 270 to separate that section.